nerdtribe/mr-binge
# Navbar and Footer

## Explanation of work done and why:


Add app navigation bar and footer and associated test placeholders.

## How to (manually) test:

1. Fetch `git fetch && git checkout -b feature/sidebar origin/feature/sidebar`
2. Serve `yarn electron:serve`
3. Observe a navbar with a logo, a button for tv and movies and the current page listed in the content area beside it. Clicking on each button toggles the view. The logo (top image) doesn't do anything when clicked.

## Checklist:
- [x]  Closes #5, Closes #12
